docs/src/pages/getting-started/support/support.md

```diff
@@ -48,24 +50,35 @@ Receive the latest news on Material-UI.
 
 ### Supported versions
 
-The Material-UI CE (Community Edition) has been 100% open-source (MIT) since the very beginning, and always will be. Developers can ensure Material-UI is the right choice for their React applications through Material-UI's community maintenance strategy. The Material-UI team regularly ships new releases, bug fixes, and is very welcoming to community pull requests.
+The MUI Core has been 100% open-source (MIT) since the very beginning in 2014, and always will be. Developers can ensure Material-UI is the right choice for their React applications through Material-UI's community maintenance strategy. The Material-UI team regularly ships new releases, bug fixes, and is very welcoming to community pull requests.
```

Comment:
MUI Core has been open-source (MIT) since the very beginning, and always will be. Developers can ensure Material-UI is the right choice for their React applications through Material-UI's community maintenance strategy. The Material-UI team regularly ships new releases, bug fixes, and is very welcoming to community pull requests.
```